internal/listeners/autovc.go

```diff
@@ -0,0 +1,114 @@
+package listeners
+
+import (
+	"github.com/bwmarrin/discordgo"
+	"github.com/sarulabs/di/v2"
+	"github.com/zekroTJA/shinpuru/internal/services/database"
+	"github.com/zekroTJA/shinpuru/internal/services/permissions"
+	"github.com/zekroTJA/shinpuru/internal/util/static"
+	"github.com/zekrotja/dgrs"
+	"log"
+	"strings"
+)
+
+var autovcCache = map[string]string{}
+var voiceStateCache = map[string]*discordgo.VoiceState{}
```

Comment:
Would make sense to add these two map instances to the `ListenerAutoVoice` struct. Doesn't make a huge difference in the logic and behavior, but 1) I am not really a fan of singleton dependencies and 2) I did it like that in every other listener and would also change it here to keep consistency. :)

Reply:
Yeah, that makes sense,
I've changed it in fefc822, is it okay that way?

Reply:
Yep, that looks fine.